### PR TITLE
DP-7022 fix fontSize in FooterHeader

### PR DESCRIPTION
**Pull request by aditudorache**
## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] The component styles are only placed in the `asc-core/src/components` folder. Exception is the `asc-ui/src/internals` folder where the styles are allowed. The components from this folder are just for internal use in the stories.
- [ ] The component style name follows the pattern `<ComponentName>Style/<ComponentName>Style.ts`
- [ ] Pull request has tests (we are going for 100% coverage!)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: packages/asc-ui/src/components/Footer/FooterContent/FooterContentStyle.ts [hunk 1]
```diff
@@ -9,5 +10,10 @@ export default styled.div<Props>`
   display: flex;
   flex-direction: column;
   margin-left: ${({ indent }) => (indent ? '28px' : '0')};
+  max-width: 303px;
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Footer/FooterContent/FooterContentStyle.ts]: plz remove? very specific, it's on the grid right?

File: packages/asc-ui/src/components/Footer/FooterHeading/FooterHeading.tsx [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+
+const FooterHeading: React.FC<
+  Props & React.HTMLAttributes<HTMLHeadingElement>
+> = ({ children, ...otherProps }) => {
+  return <FooterHeadingStyle {...otherProps}>{children}</FooterHeadingStyle>
+}
+
+FooterHeading.defaultProps = {
+  $as: 'h3',
+  styleAs: 'h3',
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Footer/FooterHeading/FooterHeading.tsx]: remove styleAs, this is optional and doesn't do anything if not defined :)

File: packages/asc-ui/src/components/Footer/FooterContent/FooterContentStyle.ts (hunk 1 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Footer/FooterContent/FooterContentStyle.ts]: this was requested from design. set on 300px

File: packages/asc-ui/src/components/Footer/FooterHeading/FooterHeading.tsx (hunk 2 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Footer/FooterHeading/FooterHeading.tsx]: removed

**Pull request merged**